docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -103,32 +97,103 @@ <h5>
+            <td>
+              Add libyaml-cpp-dev to build requirements and remove some old includes.
+            </td>
+          </tr>
+          <tr>
             <td>1158</td>
             <td><code>geeServerDefs</code> are not returned by Portable Server after being requested, once the globe has been viewed by a client. It is not possible to request <code>geeServerDefs</code> for globes other than the current <code>selectedGlobe</code>.</td>
             <td>
               <code>geeServerDefs</code> are returned correctly and can be requested for any valid globe at any time.
             </td>
           </tr>
+          <tr>
+            <td>1166</td>
+            <td>Imagery resources can be created with invalid dates</td>
+            <td>
+              Added a validity check that ensures a valid date is entered. A date of 0000-00-00 can also be entered as a default value. The user will not be allowed to save with an invalid date entered.
+            </td>
+          </tr>
+          <tr>
+            <td>1169</td>
+            <td>Cutting a globe with historical imagery embeds a link to the original Google Earth Server that served the historical imagery.</td>
+            <td>
+              There is an advanced option to include historical imagery links or not. The default option will be to NOT include links to historical imagery, and the links will be stripped from the cut globe.  When the historical imagery links are enabled, the existing behavior will be preserved.
+            </td>
+          </tr>
           <tr>
             <td>1187</td>
             <td>RPM upgrades change owner and permissions on symlinked portable globes directories</td>
             <td>
-              Symlinked portable globe directories are not modified during geserver upgrades
+              Symlinked portable globe directories are not modified during geserver upgrades.
             </td>
           </tr>
           <tr>
-            <td>1093</td>
-            <td>The <code>geselectpublishroot</code> server command line tool offers no help, and appears to take some action when the <code>--help</code> flag is specified</td>
+            <td>1190</td>
+            <td>Some 32-bit platforms are unable to open files of size >2GB</td>
             <td>
-              Help has been added through the <code>--help</code> option, or if any invalid parameter is given
+              Use POSIX file API to support large files on some 32-bit platforms.
+            </td>
+          </tr>
+          <tr>
+            <td>1192</td>
+            <td>There appears to be a regression issue in OpenJpeg 2000 lib version 2.3.0 (that Fusion uses), causing an error when fusing jp2 files.</td>
+            <td>
+               This OpenJpeg 2000 issue is fixed in OpenJpeg master but since the fix was made there was no new released version of OpenJpeg 2000 library (as per today: 2/8/2019).
```

Comment:
This is no longer accurate as OpenJpeg 2.3.1 has been released. We can change this to just say that we upgraded to OpenJpeg 2.3.1.